modules/nw-understanding-networking-dns-example.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * networking/understanding-networking.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-understanding-networking-dns-example_{context}"]
+= Example: DNS use case
+
+For this example, a front-end application is running in one set of pods and a back-end service is running in another set of pods. The front-end application needs to communicate with the back-end service. You create a service for the back-end pods that gives it a stable IP address and DNS name. The front-end pods use this DNS name to access the back-end service regardless of changes to individual pod IP addresses.
+
+By creating a service for the back-end pods, you provide a stable IP and DNS name, `backend-service.default.svc.cluster.local`, that the front-end pods can use to communicate with the back-end service. This setup would ensure that even if individual pod IP addresses change, the communication remains consistent and reliable. 
+
+.Example of configuring front-end pods to communicate with a back-end service using DNS
+
+. Create the back-end service. 
+
+.. Deploy the back-end pods.
++
+[source, yaml]
+----
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: backend-deployment
```

Comment:
Should this and the following two YAMLs have the spacing removed so that folks can copy and paste more easily?
```suggestion
----
apiVersion: apps/v1
kind: Deployment
metadata:
    name: backend-deployment
```